src/ui/components/SearchInput/style.scss

```diff
@@ -46,6 +43,15 @@ $transition-duration: 250ms;
     opacity: 1;
     // Use !important to overwrite style properties added in JS.
     transform: translateX(0) !important;
+  }
+}
+
+.SearchInput--animating {
+  .SearchInput-animation-icon {
+    transition: opacity 1ms $transition-duration, transform $transition-duration;
```

Comment:
I see `1ms` and `$transition-duration`; I didn't know about that syntax. What does it mean? I thought the third argument was the transition curve, so `duration` seems like the wrong variable.